### ISSUE TITLE
Compability with 2.4 format.

### ISSUE THREAD

**Issue by Krezalis** [bigunclemax/FTools#6]
Unsupported compressed VBF 2.4 version:
Data format identifier: 
data_format_identifier = 
0x00 = Uncompressed
0x10 = Compressed - used LZSS compression
0x01 = Encrypted ?

**Comment by bigunclemax** [comment]
Yes, VBF since v2.4 don't supported. If anyone want's to help with it I would be appreciative :smiley:

**Comment by Krezalis** [comment]
So, if we have data_format_identifier = 0x10 its mean that binary image compressed with LZSS algorithm.
And CRC calculated of unpacked image.
I think SBL unpack image inside ECU. In example full flash contained 16M, compressed 3M. Its busted time to upload FW to ECU.

**Comment by bigunclemax** [comment]
> So, if we have data_format_identifier = 0x10 its mean that binary image compressed with LZSS algorithm.

Thanks for clarify it for me. I'll try to add it soon. 
Also could you send to me any vbf with compression? (of course if you have it :)

**Comment by Krezalis** [comment]
Thanks, example: JJ5T-14C026-DB.vbf
